apps/reference/docs/architecture.mdx

```diff
@@ -2,38 +2,156 @@
-Supabase is not a 1-to-1 mapping of Firebase. While we are building many of the features that Firebase offers, we are not going about it the same way.
-
-Our technological choices are quite different from Firebase. Everything we use is open source. Wherever possible, we use and support existing tools rather than developing from scratch.
-
-Most notably, we use Postgres rather than a NoSQL store. This choice was deliberate. We believe that no other database offers the scalability and functionality required to compete with Firebase.
-
-## Feature Status
-
-| Product                    | Feature                | Stage | Docs                                             |
-| -------------------------- | ---------------------- | ----- | ------------------------------------------------ |
-| Database                   | Postgres               | GA    | [Link](/docs/guides/database)                    |
-| Database                   | Webhooks               | Alpha |                                                  |
-| Database                   | Point in time Recovery | Alpha |                                                  |
-| Realtime                   | Postgres CDC           | Beta  | [Link](/docs/guides/realtime/postgres-cdc)       |
-| Realtime                   | Broadcast              | Alpha | [Link](/docs/guides/realtime#broadcast)          |
-| Realtime                   | Presence               | Alpha | [Link](/docs/guides/realtime#presence)           |
-| Storage                    |                        | Beta  | [Link](/docs/guides/storage)                     |
-| Storage                    | CDN                    | Beta  | [Link](/docs/guides/storage-cdn)                 |
-| Edge Functions             |                        | Beta  | [Link](/docs/guides/functions)                   |
-| Auth                       | OAuth Providers        | Beta  | [Link](/docs/guides/auth/auth-apple)             |
-| Auth                       | Passwordless           | Beta  | [Link](/guides/auth/auth-magic-link)             |
-| Auth                       | Next.js Auth Helpers   | Alpha | [Link](/docs/guides/auth/auth-helpers/nextjs)    |
-| Auth                       | SvelteKit Auth Helpers | Alpha | [Link](/docs/guides/auth/auth-helpers/sveltekit) |
-| Public API                 |                        | Beta  | [Link](/docs/reference/api)                      |
-| CLI                        |                        | Beta  | [Link](/docs/guides/cli)                         |
-| Client Library: JavaScript |                        | GA    | [Link](/docs/reference/javascript)               |
-| Client Library: Dart       |                        | Beta  | [Link](/docs/reference/dart)                     |
+### PostgreSQL (Database)
+
+PostgreSQL is the core of Supabase. We do not abstract the PostgreSQL database—you can access it and use it with full privileges. We simply provide tools which makes PostgreSQL as easy to use as Firebase.
```

Comment:
just a small thing "database-you" looks like one word

Reply:
Ah, this is just a GitHub display thing. That's actually an em dash.